tests/Microsoft.Identity.Test.Integration.netfx/SeleniumTests/InteractiveFlowTests.cs

```diff
@@ -212,31 +212,51 @@ public async Task ValidateCcsHeadersForInteractiveAuthCodeFlowAsync()
 
             Assert.IsNotNull(authResult);
             Assert.IsNotNull(authResult.AccessToken);
+        }
+
+        [TestMethod]
+        public async Task Interactive_Arlington_MultiCloudSupport_AADAsync()
+        {
+            // Arrange
+            LabResponse labResponse = await LabUserHelper.GetArlingtonUserAsync().ConfigureAwait(false);
```

Comment:
Can you please create the PCA object right here in the test code, even if it leads to a small amount of duplicate code? It will make this test case more clear.